docs/streams/developer-guide/config-streams.html

```diff
@@ -243,19 +243,19 @@ <h4><a class="toc-backref" href="#id23">num.standby.replicas</a><a class="header
           </tr>
           </thead>
           <tbody valign="top">
-          <tr class="row-even"><td>acceptable.recovery.lag</td>
+          <tr class="row-odd"><td>acceptable.recovery.lag</td>
```

Comment:
Why do we change this? Seem the top row with "Parameter Name" is `odd` already, so `even` seems to be correct here?